sugondat/chain/node/src/cli.rs

```diff
@@ -34,7 +34,7 @@ pub enum Subcommand {
     ExportGenesisMetadata(export_genesis_metadata::ExportGenesisMetadataCmd),
 
     /// Export the genesis state of the parachain.
-    ExportGenesisState(cumulus_client_cli::ExportGenesisStateCommand),
+    ExportGenesisHead(cumulus_client_cli::ExportGenesisHeadCommand),
```

Comment:
I know that the documentation above is not really informative, but perhaps should be updated for consistency.